src/pages/questionPage.js

```diff
@@ -70,7 +71,13 @@ export const initQuestionPage = () => {
 };
 
 const nextQuestion = () => {
-  quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
-  setQuizData(quizData);
-  initQuestionPage();
+  //>>>LAST PAGE >>>
+  if (quizData.currentQuestionIndex < 9) {
```

Comment:
Instead of hard-coded number can you use `quizData.questions.length - 1`

Reply:
done!